explorer/src/components/Nav.tsx

```diff
@@ -112,51 +111,67 @@ const navOptions: navOptionType[] = [
   {
     url: '/overview',
     title: 'Overview',
-    Icon: <BarChart />,
+    renderIcon: (mode) => <OverviewSVG mode={mode} />
```

Comment:
I would just call this Icon and pass through the component `<OverviewSVG />`. When I'm passing a component as a prop I usually uppercase the first letter of the prop name (in this can Icon). It helps to quickly identify props that are components. Just something I do and only a suggestion.